drivers/gles3/rasterizer_scene_gles3.cpp

```diff
@@ -2908,13 +2908,13 @@ void RasterizerSceneGLES3::_setup_reflections(RID *p_reflection_probe_cull_resul
 			reflection_ubo.ambient[3] = rpi->probe_ptr->interior_ambient_probe_contrib;
 		} else {
 			Color ambient_linear;
-			float contrib = 0;
+			//float contrib = 0;
```

Comment:
Please don't keep commented out code, it's pointless if nothing uses it.

Reply:
I kept a couple where I wasn't sure if it was just missed or if it is truly unused. I can fully remove the lines too.

Reply:
Well if a search in the file shows no usage occurrence, just remove it. If it shows it's used somewhere, it would be in commented out code, in which case we need to assess if it's meant to be removed or kept for future reimplementation.